.claude/skills/e2e-test/EXAMPLES.md

```
@@ -0,0 +1,256 @@
+1. Ask you to select features (multi-select enabled)
+2. Ask you to select OS platform
+3. Ask for bundle location (~/.crc/cache, ~/Downloads, or custom)
+4. Ask for pull secret location (~/Downloads or custom)
+5. Build CRC from source using `make cross` (~5-10 minutes)
+6. Verify prerequisites
+7. Build and execute the test command with the built binary
+8. Show results
+
+### Example Interactive Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)